### PR TITLE
[DATAHUBIO-57-arbitrary-units]

### PR DESCRIPTION
**Pull request by StencelMichal**
Visualization of units and values with proper number of decimal places. Changes in config file

### CODE REVIEW

File: src/chart/axisSide.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class AxisSide {
+    static LEFT = new AxisSide("LEFT")
+    static RIGHT = new AxisSide("RIGHT")
+
+    constructor(side) {
+        side = side.toLowerCase()
+        if (side !== "left" && side !== "right") {
+            throw new Error("side != left or right")
```
Comment (Singularity23x0) [inline src/chart/axisSide.js]: Says "side not eq left OR right", but the if above states "not equal left AND not equal right".

File: src/chart/chart.js [hunk 2]
```diff
@@ -15,22 +17,27 @@ class Chart {
     }
 
     addData(newData) {
+        newData.forEach(v =>
+            Object.entries(v.values).forEach(([name, value]) => {
+                const side = this.dataSeries[name].yAxisSide;
+                const axis = side === AxisSide.LEFT ? this.metadata.leftAxis : this.metadata.rightAxis;
```
Comment (Singularity23x0) [inline src/chart/chart.js:23]: I propose splitting this line due to its length.
The format is:
<condition>
? <variant A>
: <variant B>;

File: src/components/plot.jsx [hunk 3]
```diff
@@ -1,22 +1,44 @@
...
                     <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
                     <XAxis dataKey={"timestamp"} hide={false}/>
-                    <YAxis/>
                     {
                         Object.entries(chart.dataSeries).map(dataSeries => {
-                            return (<Line key={dataSeries[0]}
+                            const seriesName = dataSeries[0];
+                            const seriesInfo = dataSeries[1];
+                            const yAxisSide = seriesInfo.yAxisSide;
+                            const axis = yAxisSide.value === AxisSide.LEFT.value ? leftAxis : rightAxis;
```
Comment (Singularity23x0) [inline src/components/plot.jsx:32]: I again propose splityibg the condidon operator into 3 lines

**Review by Singularity23x0 — CHANGES_REQUESTED** [review]
Changes proposed in comments.

File: src/chart/chart.js (hunk 2 above)
Comment (Singularity23x0) [reply src/chart/chart.js:23]: condidtion
? variantA
: variantB;

Comment (StencelMichal) [reply src/chart/chart.js:23]: That's what ternary operator is for - to inline if statements

File: src/components/plot.jsx (hunk 3 above)
Comment (StencelMichal) [reply src/components/plot.jsx:32]: Again that's what ternary operator is for - to inline if statements

**Pull request merged**